src/frost.rs

```diff
@@ -63,17 +63,19 @@ impl From<jubjub::ExtendedPoint> for Public {
 #[derive(Clone)]
 pub struct Share {
     receiver_index: u64,
```

Comment:
At some point we should probably update the names of these `u64`s from `index` to `id`, can you open a ticket for that?

Reply:
https://github.com/ZcashFoundation/redjubjub/issues/116